docs/content/customization/nutanix/prism-central-endpoint.md

```diff
@@ -43,3 +43,43 @@ spec:
           kind: Secret
           name: secret-name
 ```
+
+### Provide an Optional Trusted CA Bundle
+
+If the Prism Central endpoint uses a self-signed certificate, you can provide an additional trust bundle
+to be used by the Nutanix provider.
+This is a base64 PEM encoded x509 cert for the RootCA that was used to create the certificate for a Prism Central
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: <NAME>
+spec:
+  topology:
+    variables:
+      - name: clusterConfig
+        value:
+          nutanix:
+            prismCentralEndpoint:
+              # ...
+              additionalTrustBundle: "LS0...="
```

Comment:
should we also point here for more details in this doc https://opendocs.nutanix.com/capx/v1.3.x/pc_certificates/ and https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Security-Guide-v6_5:mul-security-ssl-certificate-pc-t.html for more details.